man/dracut.conf.5.asc

```diff
@@ -142,8 +142,8 @@ Configuration files must have the extension .conf; other extensions are ignored.
 
 *persistent_policy=*"__<policy>__"::
     Use _<policy>_ to address disks and partitions.
-    _<policy>_ can be any directory name found in /dev/disk.
-    E.g. "by-uuid", "by-label"
+    _<policy>_ can be any directory name found in /dev/disk (e.g. "by-uuid",
+    "by-label"), or "mapper" to use /dev/mapper device names (default=mapper).
```

Comment:
This man page is for all dracut deployment. Is "mapper" the default for all dracut installations ?